tests/testproject/TestProject.AppHost/TestProject.AppHost.csproj

```diff
@@ -8,39 +8,22 @@
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.Azure.EventHubs\Aspire.Hosting.Azure.EventHubs.csproj" IsAspireProjectResource="false" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.AppHost\Aspire.Hosting.AppHost.csproj" IsAspireProjectResource="false" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.Garnet\Aspire.Hosting.Garnet.csproj" IsAspireProjectResource="false" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.Milvus\Aspire.Hosting.Milvus.csproj" IsAspireProjectResource="false" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.MongoDB\Aspire.Hosting.MongoDB.csproj" IsAspireProjectResource="false" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.MySql\Aspire.Hosting.MySql.csproj" IsAspireProjectResource="false" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.NodeJs\Aspire.Hosting.NodeJs.csproj" IsAspireProjectResource="false" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.Oracle\Aspire.Hosting.Oracle.csproj" IsAspireProjectResource="false" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.PostgreSQL\Aspire.Hosting.PostgreSQL.csproj" IsAspireProjectResource="false" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.RabbitMQ\Aspire.Hosting.RabbitMQ.csproj" IsAspireProjectResource="false" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.Redis\Aspire.Hosting.Redis.csproj" IsAspireProjectResource="false" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.SqlServer\Aspire.Hosting.SqlServer.csproj" IsAspireProjectResource="false" />
-    <ProjectReference Include="..\..\..\src\Aspire.Hosting.Valkey\Aspire.Hosting.Valkey.csproj" IsAspireProjectResource="false" />
-    
-  </ItemGroup>
-  <ItemGroup Condition="'$(TestsRunningOutsideOfRepo)' == 'true'">
-    <PackageReference Include="Aspire.Hosting.Azure" />
-    <PackageReference Include="Aspire.Hosting.Azure.CosmosDB" />
-    <PackageReference Include="Aspire.Hosting.Azure.EventHubs" />
-    <PackageReference Include="Aspire.Hosting.AppHost" />
-    <PackageReference Include="Aspire.Hosting.Garnet" />
-    <PackageReference Include="Aspire.Hosting.Milvus" />
-    <PackageReference Include="Aspire.Hosting.MongoDB" />
-    <PackageReference Include="Aspire.Hosting.MySql" />
-    <PackageReference Include="Aspire.Hosting.NodeJs" />
-    <PackageReference Include="Aspire.Hosting.Oracle" />
-    <PackageReference Include="Aspire.Hosting.PostgreSQL" />
-    <PackageReference Include="Aspire.Hosting.RabbitMQ" />
-    <PackageReference Include="Aspire.Hosting.Redis" />
-    <PackageReference Include="Aspire.Hosting.SqlServer" />
-    <PackageReference Include="Aspire.Hosting.Valkey" />
+  <ItemGroup>
+    <NonComponentReferenceForTests Include="Aspire.Hosting.Azure" IsAspireProjectResource="false" />
```

Comment:
Why is this named NonComponentReferenceForTests? What does "Component" matter?

Reply:
Lack of a better name! Please suggest one :)

Reply:
Maybe drop `Component` and just have `ReferenceForTests`. Or `OutsideOfRepoReference`?

Reply:
After some offline discussion with @eerhardt I was able to combine the two into a single item `AspireProjectOrPackageReference`. Feel free to suggest a different name though :)